tools/misc/labelme2coco.py

```diff
@@ -0,0 +1,322 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import argparse
+import glob
+import json
+import os
+import shutil
+
+import numpy as np
+from sklearn.model_selection import train_test_split
+from tqdm import tqdm
+
+# import sys
+
+
+class Labelme2coco_keypoints():
+
+    def __init__(self, args):
+        """Lableme 关键点数据集转 COCO 数据集的构造函数:
```

Comment:
How about translating these sentences into English?

Reply:
I don't have much time these days because of my studies,could you please help me?